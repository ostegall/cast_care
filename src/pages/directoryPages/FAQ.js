import { Text, View } from 'react-native';
import styles from '../../styles';

export default function FAQ({ }) {
  //Page for answering common questions about cast upkeep as well as for information about application
  return (
    <View style={{ flex: 1, paddingTop: 20 }}>
      <Text style={styles.title_text}>
        FAQ
      </Text>
      <Text style={styles.page_text}>
        What should I do if my cast issue is not listed in the app?
      </Text>
      <Text style={styles.page_text}>
        {'\u2022'} If you feel as though you have an issue with your cast that does not fall under one of the categories we cover here, please contact your provider.
      </Text>
      <Text style={styles.page_text}>
        Does this app collect and store any data about my personal information or health?
      </Text>
      <Text style={styles.page_text}>
        {'\u2022'} No, this app does not collect or store any data about its users.
      </Text>
      <Text style={styles.page_text}>
        What should I do if I was treated at a non-UNC hospital?
      </Text>
      <Text style={styles.page_text}>
        {'\u2022'} The information in the app would still apply, except you do not have to go to a UNC hospital as the app suggests
      </Text>
      <View style={{ paddingTop: 10 }}>
        <Text style={styles.title_text}>
          About
        </Text>
        <Text style={styles.page_text}>
          CastCare is a mobile information and support system for patients who are having issues with their casts or are interested in learning more about taking proper care of their casts. By providing general instructions in a clear and simple manner, CastCare hopes to provide users with essential at-home access to medical information that is often contained in physical pamphlets.
        </Text>
      </View>
    </View>
  )
}