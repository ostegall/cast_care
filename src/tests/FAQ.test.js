import renderer from 'react-test-renderer';
import { shallow } from 'enzyme'

import FAQ from '../pages/directoryPages/FAQ';

//Makes sure app component renders correctly without crashing
it("renders without crashing", () => {
  shallow(<FAQ />);
});

//Snapshot test to ensure UI stays consistent
it('renders correctly', () => {
  const tree = renderer.create(<FAQ />).toJSON();
  expect(tree).toMatchSnapshot();
});