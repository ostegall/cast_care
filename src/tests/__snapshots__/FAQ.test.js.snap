// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "paddingTop": 20,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 23,
        "fontWeight": "bold",
        "letterSpacing": 0.5,
        "textAlign": "center",
      }
    }
  >
    FAQ
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "fontSize": 17,
        "letterSpacing": 0.5,
        "marginBottom": 5,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 5,
      }
    }
  >
    What should I do if my cast issue is not listed in the app?
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "fontSize": 17,
        "letterSpacing": 0.5,
        "marginBottom": 5,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 5,
      }
    }
  >
    •
     If you feel as though you have an issue with your cast that does not fall under one of the categories we cover here, please contact your provider.
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "fontSize": 17,
        "letterSpacing": 0.5,
        "marginBottom": 5,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 5,
      }
    }
  >
    Does this app collect and store any data about my personal information or health?
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "fontSize": 17,
        "letterSpacing": 0.5,
        "marginBottom": 5,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 5,
      }
    }
  >
    •
     No, this app does not collect or store any data about its users.
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "fontSize": 17,
        "letterSpacing": 0.5,
        "marginBottom": 5,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 5,
      }
    }
  >
    What should I do if I was treated at a non-UNC hospital?
  </Text>
  <Text
    style={
      Object {
        "alignItems": "center",
        "fontSize": 17,
        "letterSpacing": 0.5,
        "marginBottom": 5,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 5,
      }
    }
  >
    •
     The information in the app would still apply, except you do not have to go to a UNC hospital as the app suggests
  </Text>
  <View
    style={
      Object {
        "paddingTop": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 23,
          "fontWeight": "bold",
          "letterSpacing": 0.5,
          "textAlign": "center",
        }
      }
    >
      About
    </Text>
    <Text
      style={
        Object {
          "alignItems": "center",
          "fontSize": 17,
          "letterSpacing": 0.5,
          "marginBottom": 5,
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 5,
        }
      }
    >
      CastCare is a mobile information and support system for patients who are having issues with their casts or are interested in learning more about taking proper care of their casts. By providing general instructions in a clear and simple manner, CastCare hopes to provide users with essential at-home access to medical information that is often contained in physical pamphlets.
    </Text>
  </View>
</View>
`;
